CHANGELOG.md

```diff
@@ -22,17 +22,29 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+### Breaking changes
+
+-  Localization
+   -  `locale` prop: `zh-YUE` has been renamed to `yue` to conform with Unicode standard. `zh-YUE` will continue to work with warnings
+   -  Most strings are being retranslated by Microsoft localization team, except English (US), Egyptian Arabic, and Chinese Yue
+      -  If the new strings are undesirable, please use the [`overideLocalizedStrings` prop](https://github.com/microsoft/BotFramework-WebChat/tree/master/docs/LOCALIZATION.md#overriding-localization-strings) to override
+      -  String IDs are being refreshed and use a standard format
+   -  `useLocalize` and `useLocalizeDate` is being deprecated. Please use `useLocalizer` and `useDateFormatter` instead
```

Comment:
don't use 'is being'. Means it's still in the process of happening. When this is published, it will be done.
Link to documentation for `useLocalizer` and `useDateFormatter`